.github/workflows/cpp-packaging.yml

```diff
@@ -561,8 +561,8 @@ jobs:
               # MacOS: use LLVM binutils for both X64 and ARM64
               additional_flags+=(-L)
             elif [[ "${{ matrix.sdk_platform }}" == "windows" && "${variant}" == *"/x64/"* ]]; then
-              # Windows x64: force input and output target format
-              additional_flags+=(-f pe-x86-64,pe-bigobj-x86-64)
+              # Windows x64: force input and output target format and use LLVM binutils for correct objcopy (issue #793)
```

Comment:
Rather than installing llvm-objcopy from binary, I'd rather compile it from source as we do on Mac above.

Reply:
Makes sense. Not why we're building with `-DLLVM_ENABLE_PROJECTS=clang` though, `-DLLVM_ENABLE_PROJECTS=llvm` would be enough since only llvm binaries are being built.

Reply:
Happy to change this as needed as I’m not sure I fully get the whole setup, but what’s the benefit of building these LLVM tools from source vs. simply installing the LLVM package in `install_prereqs_desktop.py`?

Reply:
Because this way we can be sure exactly which binary formats the objcopy command supports. In some cases, objcopy on platforms only supports a subset of binary formats (which would be relevant to the platform). By building it ourselves, we can be sure that it supports all of the binary formats we need.

Reply:
Thanks, I’ve changed it so it should be using self-built LLVM binutils.